doc/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -j auto -W --keep-going -w build_errors.txt -N -q
+SPHINXOPTS    = -j auto --keep-going -w build_errors.txt -N -q
```

Comment:
Why did you turn off warnings?

Reply:
Warnings turns into the error and results into doc build failure.
 
 We are getting more than 25000 warnings hence we have turned off all warnings.